crates/sui-sdk/src/lib.rs

```diff
@@ -197,25 +400,30 @@ impl Debug for RpcClient {
     }
 }
 
+/// ServerInfo contains all the useful information regarding the API version, the available RPC calls, and subscriptions.
 struct ServerInfo {
     rpc_methods: Vec<String>,
     subscriptions: Vec<String>,
     version: String,
 }
 
 impl SuiClient {
+    /// Returns a list of available RPC calls.
     pub fn available_rpc_methods(&self) -> &Vec<String> {
         &self.api.info.rpc_methods
     }
 
+    /// Returns a list of available subscriptions.
```

Comment:
Needs a little bit more detail:

```suggestion
    /// Returns a list of streaming/subscription APIs supported by the node the client is connected to.
```